packages/core/src/components/Tabs/Tab/Tab.tsx

```diff
@@ -77,25 +80,27 @@ const Tab: FC<TabProps> = forwardRef(
       return [...childrenArray, iconElement];
     }
     return (
-      <li
-        ref={mergedRef}
-        key={id}
-        className={cx(styles.tabWrapper, className, {
-          [styles.active]: active,
-          [styles.disabled]: disabled,
-          [styles.tabFocusVisibleInset]: focus
-        })}
-        id={id}
-        role="tab"
-        aria-selected={active}
-        aria-disabled={disabled}
-        data-testid={dataTestId || getTestId(ComponentDefaultTestId.TAB, id)}
-      >
-        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid,jsx-a11y/click-events-have-key-events */}
-        <a className={cx(styles.tabInner, tabInnerClassName)} onClick={() => !disabled && onClick(value)}>
-          {renderIconAndChildren()}
-        </a>
-      </li>
+      <Tooltip {...tooltipProps} content={disabled && tooltipProps.content}>
```

Comment:
The tooltip props/content should not be conditioned by the disabled, but should be allowed for general tooltips as well (which means that the user of the component should condition the tooltip according to the disable value)